app/build.gradle

```diff
@@ -39,13 +44,15 @@ dependencies {
     compile 'org.encryptor4j:encryptor4j:0.1.1'
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:2.8.9'
-    testCompile 'org.robolectric:robolectric:3.0'
+    testCompile "org.robolectric:robolectric:3.0"
```

Comment:
We first tried another method for importing the database, and this required a different version of robolectric. Since we chose a different method, we set the version back to how it originally was. Don't know why GitHub shows this.